packages/ergo-compiler/lib/fileloader.js

```diff
@@ -90,16 +90,21 @@ class FileLoader {
      * @param {string} fileName the relative file name
      * @param {boolean} json if true the file is converted to a JS Object using JSON.parse
      * @param {boolean} required whether the file is required
+     * @param {Buffer} buffer if true the file is converted to a Buffer
      * @return {Promise<string>} a promise to the contents of the file or null if it does not exist and
      * required is false
      */
-    static async loadFileContents(path, fileName, json=false, required=false) {
+    static async loadFileContents(path, fileName, json=false, required=false, buffer=false) {
```

Comment:
I think I'd rather have a separate `loadFileBuffer` method, which wouldn't need a `json` parameter (which is ignored in this case). It might be useful to have a way to cap the size of the buffer (throwing an error for files above a certain size?).

Reply:
@jeromesimeon What size of image do you recommend? How many KBs approx?

Reply:
> @jeromesimeon What size of image do you recommend? How many KBs approx?

I'm not sure. I would raise this question in the issue and Slack Cicero channel.

Reply:
What's a typical size for a 256x256 image in format XXX?